### PR TITLE
[release_v300] release notes template

### PR DESCRIPTION
**Pull request by AlexanderDokuchaev**
### Reason for changes

Upcoming release

### Related tickets

CVS-180374

### CODE REVIEW

File: ReleaseNotes.md [hunk 1]
```diff
@@ -1,5 +1,56 @@
...
+- Breaking changes:
+  - ...
+- General:
+  - ...
+- Features:
+  - (TorchFX) Preview support for the new `compress_pt2e` API has been introduced, enabling quantization of `torch.fx.GraphModule` models with the `OpenVINOQuantizer`. Users now can quantize their models in [ExecuTorch](https://github.com/pytorch/executorch) for the OpenVINO backend via the nncf `compress_pt2e` employing Scale Estimation and AWQ.
+- Fixes:
+  - ...
+- Improvements:
+  - Add Support for compression of 3D weights in AWQ, Scale Estimation and GPTQ Algorithms. Models with MoE can be compressed now.
```
Comment (anzr299) [inline ReleaseNotes.md]: #3728 #3706 #3835

File: ReleaseNotes.md [hunk 2]
```diff
@@ -1,5 +1,56 @@
...
+## New in Release 3.0.0
+
+Post-training Quantization:
+
+- Breaking changes:
+  - ...
+- General:
+  - ...
+- Features:
+  - (TorchFX) Preview support for the new `compress_pt2e` API has been introduced, enabling quantization of `torch.fx.GraphModule` models with the `OpenVINOQuantizer`. Users now can quantize their models in [ExecuTorch](https://github.com/pytorch/executorch) for the OpenVINO backend via the nncf `compress_pt2e` employing Scale Estimation and AWQ.
```
Comment (anzr299) [inline ReleaseNotes.md:18]: #3663

File: ReleaseNotes.md [hunk 3]
```diff
@@ -1,5 +1,57 @@
...
+## New in Release 3.0.0
+
+Post-training Quantization:
+
+- Breaking changes:
+  - ...
+- General:
+  - ...
+- Features:
+  - (OpenVINO) Introduced new experimental compression data type ADAPTIVE_CODEBOOK. This compression type calculates a unique codebook for each MatMul or block of identical MatMuls (for example, all down_proj could have the same codebook). This approach reduces quality degradation in the case of per-channel weight compression.
```
Comment (andreyanufr) [inline ReleaseNotes.md]: https://github.com/openvinotoolkit/nncf/pull/3684

File: ReleaseNotes.md [hunk 4]
```diff
@@ -1,5 +1,58 @@
...
+- Features:
+  - ...
+- Fixes:
+  - ...
+- Improvements:
+  - ...
+- Deprecations/Removals:
+  - ...
+- Tutorials:
+  - [Post-Training Quantization of YOLO26 OpenVINO Model](https://github.com/openvinotoolkit/nncf/tree/develop/examples/post_training_quantization/openvino/yolo26)
```
Comment (daniil-lyakhov) [inline ReleaseNotes.md]: https://github.com/openvinotoolkit/nncf/pull/3844

File: ReleaseNotes.md [hunk 5]
```diff
@@ -1,5 +1,58 @@
...
+## New in Release 3.0.0
+
+Post-training Quantization:
+
+- Breaking changes:
+  - ...
+- General:
+  - ...
+- Features:
+  - Models containing MatMul operations with transposed activation inputs was supported in Weight Compression and AWQ algorithms.
```
Comment (daniil-lyakhov) [inline ReleaseNotes.md]: https://github.com/openvinotoolkit/nncf/pull/3794

File: ReleaseNotes.md [hunk 6]
```diff
@@ -1,5 +1,59 @@
...
+  - (OpenVINO) Introduced new experimental compression data type ADAPTIVE_CODEBOOK. This compression type calculates a unique codebook for each MatMul or block of identical MatMuls (for example, all down_proj could have the same codebook). This approach reduces quality degradation in the case of per-channel weight compression.
+  - (TorchFX) Preview support for the new `compress_pt2e` API has been introduced, enabling quantization of `torch.fx.GraphModule` models with the `OpenVINOQuantizer`. Users now can quantize their models in [ExecuTorch](https://github.com/pytorch/executorch) for the OpenVINO backend via the nncf `compress_pt2e` employing Scale Estimation and AWQ.
+- Fixes:
+  - (ONNX) Fixed `compress_quantize_weights_transformation()` method by removing names of deleted initializers from graph inputs.
```
Comment (andrey-churkin) [inline ReleaseNotes.md:22]: https://github.com/openvinotoolkit/nncf/pull/3885

File: ReleaseNotes.md [hunk 7]
```diff
@@ -1,5 +1,59 @@
...
+  - (ONNX) Fixed incorrect insertion of MatMulNBits nodes.
```
Comment (andrey-churkin) [inline ReleaseNotes.md:23]: https://github.com/openvinotoolkit/nncf/pull/3889

File: ReleaseNotes.md [hunk 8]
```diff
@@ -1,5 +1,69 @@
 # Release Notes
 
+## New in Release 3.0.0
+
+Post-training Quantization:
+
+- Breaking changes:
+  - (TensorFlow) Removed support for TensorFlow backend.
+  - (PyTorch) Removed legacy `create_compressed_model` API for PyTorch backend, which was previously marked as deprecated.
+  - (PyTorch) Removed legacy algorithms for PyTorch that were based on using `NNCFNetwork`, e.g. NAS, Structural Pruning, AutoML, Knowledge Distillation, Mixed-Precision Quantization.
```
Comment (MaximProshin) [inline ReleaseNotes.md]: Let's avoid "e.g." here and list all removed methods. If it's the full list, just replace ", e.g." by ":".

File: ReleaseNotes.md [hunk 9]
```diff
@@ -1,5 +1,69 @@
...
+  - Renamed `nncf.CompressWeightsMode.CB4_F8E4M3` mode option to `nncf.CompressWeightsMode.CB4`.
+- General:
+  - Added `nncf.prune` API function, which provides a unified interface for pruning algorithms. Currently available for PyTorch backend and supports Magnitude Pruning.
+    More details about the new API can be found in the [documentation](https://github.com/openvinotoolkit/nncf/tree/develop/docs/usage/training_time_compression/pruning/Usage.md).
+  - Added `nncf.build_graph` API function for building `NNCFGraph` from a model.
```
Comment (MaximProshin) [inline ReleaseNotes.md]: Can you please add how and when it could be used? We need some explanation of why it was added.

File: ReleaseNotes.md [hunk 10]
```diff
@@ -1,5 +1,69 @@
...
+  - Added [documentation](https://github.com/openvinotoolkit/nncf/blob/develop/docs/usage/IgnoredScope.md) about using `nncf.IgnoredScope`.
+  - Reworked `HWConfig`, now using Python-style definition of hardware configuration instead of JSON files.
+- Features:
+  - Models containing MatMul operations with transposed activation inputs was supported in Weight Compression and AWQ algorithms.
```
Comment (MaximProshin) [inline ReleaseNotes.md]: I would rephrase it to "Added support for models containing MatMul operations with transposed activation inputs in data-free Weight Compression and data-aware AWQ algorithms."

File: ReleaseNotes.md [hunk 11]
```diff
@@ -1,5 +1,69 @@
...
+  - (TorchFX) Preview support for the new `compress_pt2e` API has been introduced, enabling quantization of `torch.fx.GraphModule` models with the `OpenVINOQuantizer`. Users now can quantize their models in [ExecuTorch](https://github.com/pytorch/executorch) for the OpenVINO backend via the nncf `compress_pt2e` employing Scale Estimation and AWQ.
+  - (PyTorch) Added support for linear functions for the Fast Bias Correction algorithm.
+  - (OpenVINO) Added [activation profiler](https://github.com/openvinotoolkit/nncf/tree/develop/tools/activation_profiler) tool to collect and visualize tensor statistics.
+- Fixes:
+  - (ONNX) Fixed `compress_quantize_weights_transformation()` method by removing names of deleted initializers from graph inputs.
+  - (ONNX) Fixed incorrect insertion of MatMulNBits nodes.
+- Improvements:
+  - Add Support for compression of 3D weights in AWQ, Scale Estimation and GPTQ Algorithms. Models with MoE (Mixture of Experts) can be compressed now.
```
Comment (MaximProshin) [inline ReleaseNotes.md]: I would rephrase it to "Added support for the compression of 3D weights in AWQ, Scale Estimation, and GPTQ algorithms. Models with MoE (Mixture of Experts), such as GPT-OSS-20B and Qwen3-30B-A3B, can be compressed with data-aware methods now."

File: ReleaseNotes.md [hunk 12]
```diff
@@ -1,5 +1,69 @@
...
+- Deprecations/Removals:
+  - ...
+- Tutorials:
```
Comment (MaximProshin) [inline ReleaseNotes.md:26]: Please also add: https://github.com/openvinotoolkit/openvino_notebooks/tree/latest/notebooks/funasr-nano
and https://github.com/openvinotoolkit/openvino_notebooks/tree/latest/notebooks/cosyvoice3-tts

File: ReleaseNotes.md [hunk 13]
```diff
@@ -1,5 +1,69 @@
...
+- General:
+  - Added `nncf.prune` API function, which provides a unified interface for pruning algorithms. Currently available for PyTorch backend and supports Magnitude Pruning.
+    More details about the new API can be found in the [documentation](https://github.com/openvinotoolkit/nncf/tree/develop/docs/usage/training_time_compression/pruning/Usage.md).
+  - Added `nncf.build_graph` API function for building `NNCFGraph` from a model.
+  - Added [documentation](https://github.com/openvinotoolkit/nncf/blob/develop/docs/usage/IgnoredScope.md) about using `nncf.IgnoredScope`.
+  - Reworked `HWConfig`, now using Python-style definition of hardware configuration instead of JSON files.
+- Features:
+  - Models containing MatMul operations with transposed activation inputs was supported in Weight Compression and AWQ algorithms.
+  - (OpenVINO) Introduced new experimental compression data type ADAPTIVE_CODEBOOK. This compression type calculates a unique codebook for each MatMul or block of identical MatMuls (for example, all down_proj could have the same codebook). This approach reduces quality degradation in the case of per-channel weight compression.
```
Comment (MaximProshin) [inline ReleaseNotes.md]: Let's add a link to https://github.com/openvinotoolkit/nncf/tree/develop/examples/llm_compression/openvino/smollm2_360m_adaptive_codebook

File: ReleaseNotes.md [hunk 14]
```diff
@@ -1,5 +1,69 @@
...
+  - (TorchFX) Preview support for the new `compress_pt2e` API has been introduced, enabling quantization of `torch.fx.GraphModule` models with the `OpenVINOQuantizer`. Users now can quantize their models in [ExecuTorch](https://github.com/pytorch/executorch) for the OpenVINO backend via the nncf `compress_pt2e` employing Scale Estimation and AWQ.
+  - (PyTorch) Added support for linear functions for the Fast Bias Correction algorithm.
```
Comment (MaximProshin) [inline ReleaseNotes.md]: Is it clear enough? Can we add some model examples?

File: ReleaseNotes.md [hunk 15]
```diff
@@ -1,5 +1,69 @@
...
+- Features:
+  - ...
+- Fixes:
+  - ...
+- Improvements:
+  - ...
+- Deprecations/Removals:
+  - ...
+- Tutorials:
+  - [Post-Training Quantization of YOLO26 OpenVINO Model](https://github.com/openvinotoolkit/nncf/tree/develop/examples/post_training_quantization/openvino/yolo26)
```
Comment (MaximProshin) [inline ReleaseNotes.md]: Why is it in QAT section?

**Review by MaximProshin — CHANGES_REQUESTED** [review]

File: ReleaseNotes.md (hunk 11 above)
Comment (anzr299) [reply ReleaseNotes.md]: Alright, done

File: ReleaseNotes.md (hunk 12 above)
Comment (l-bat) [reply ReleaseNotes.md:26]: done

File: ReleaseNotes.md (hunk 10 above)
Comment (daniil-lyakhov) [reply ReleaseNotes.md]: Done

File: ReleaseNotes.md (hunk 15 above)
Comment (daniil-lyakhov) [reply ReleaseNotes.md]: Moved to PTQ

File: ReleaseNotes.md (hunk 9 above)
Comment (AlexanderDokuchaev) [reply ReleaseNotes.md]: Suggestion to use it for setup [IgnoreScope](https://github.com/openvinotoolkit/nncf/blob/develop/docs/usage/IgnoredScope.md)

File: ReleaseNotes.md (hunk 14 above)
Comment (AlexanderDokuchaev) [reply ReleaseNotes.md]: I think yes,
In our tests scope it's not provide any changing metrics
For example resnet18 use it https://github.com/pytorch/vision/blob/main/torchvision/models/resnet.py#L206 that used in examples and PTQ

File: ReleaseNotes.md (hunk 9 above)
Comment (MaximProshin) [reply ReleaseNotes.md]: Can you specify it in RN then? Do we have an example, docs or smth we could refer to?

File: ReleaseNotes.md (hunk 14 above)
Comment (MaximProshin) [reply ReleaseNotes.md]: Ok. I suppose it means that the accuracy of such models could be improved and if so, I suggest to extend it to "(PyTorch) Added support for linear functions for the Fast Bias Correction algorithm to improve the accuracy of such models after the quantization."
